corehq/apps/hqwebapp/templates/hqwebapp/partials/ko_inline_edit.html

```diff
@@ -48,5 +48,5 @@
         </button>
       </div>
     </div>
-  </div><span data-bind="template: {afterRender: afterRenderFunc}"></span>
+  </div><span data-bind="template: {afterRender: afterRenderFunc}">&nbsp;</span> {# please keep the &nbsp; to satisfy the knockout 3.5.1 gods #}
```

Comment:
Do you know which part requires the `&nbsp;`? I'm wondering if this problem could affect other usages of `template` or specifically `afterRender`.